### DISCUSSION TITLE
añadir puntuación de filmaffinity

### DISCUSSION THREAD

**Discussion by Hoskargl** [smtdev/tautulli-telegram-notifications#6] [Ideas]
Me gustaría saber si es posible que añadas dentro de la plantilla la puntuación que otorga filmaffinity. Personalmente me parece más ajustadas las puntuaciones en filmaffinity que en IMDB.
Gracias.
Un saludo.

**Comment by smtdev** [comment]
Buenas,

Lamentablemente no es posible, ya que Tautulli no las contempla como una de sus variables disponibles. Tal vez podrías abrir una petición en el propio repositorio de Tautulli, aunque te diré que me temo que FilmAffinity no tiene API, no pública/oficial al menos, por lo que será complicado.

**Comment by Hoskargl** [comment]
Muchas gracias por la información y por contestar tan rápido.
Un saludo.
Óscar.

Enviado desde Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
